cou/apps/auxiliary.py

```diff
@@ -33,6 +33,31 @@
 class OpenStackAuxiliaryApplication(OpenStackApplication):
     """Application for charms that can have multiple OpenStack releases for a workload."""
 
+    @property
+    def is_os_channel_based(self) -> bool:
+        """Check if application is OpenStack channel based.
+
+        For auxiliary charms, always return false because they are
+        not OpenStack channel based.
+        :return: False.
+        :rtype: bool
+        """
+        return False
+
+    def is_valid_track(self, charm_channel: str) -> bool:
+        """Check if the channel track is valid.
+
+        :param charm_channel: Charm channel. E.g: 3.8/stable
+        :type charm_channel: str
+        :return: True if valid, False otherwise.
+        :rtype: bool
+        """
+        if self.is_from_charm_store:
+            return True
```

Comment:
I think it would be good if we provide debug log message that valid track is selected because charm comes from cs.